test/include/RAJA_test-multi-reducepol.hpp

```diff
@@ -43,7 +43,7 @@ using HipMultiReducePols =
               RAJA::hip_multi_reduce_global_atomic_no_replication_host_init  >;
 #endif
 
-#if defined(RAJA_ENABLE_SYCL)
+#if defined(RAJA_ENABLE_SYCL) && 0
```

Comment:
Is this needed? It looks like the SYCL tests are disabled in the CMakeLists.txt file in the test directory.

Reply:
yes, when sycl is enabled and this header is included in tests for other backends this causes an error as sycl_multi_reduce does not exist.

Reply:
I could also just delete it for now.

Reply:
The fact that it causes compilation issues seems odd. Does the type not exist, or is it guarded in a way that differs from other backends? We don't have that problem with other tests?

Reply:
I removed this because the type doesn't exist.